Source/WebCore/style/PropertyCascade.cpp

```diff
@@ -106,6 +108,30 @@ void PropertyCascade::buildCascade()
     sortLogicalGroupPropertyIDs();
 }
 
+void PropertyCascade::injectUnsetForZoomAffectedProperties()
+{
+    static const CSSPropertyID zoomAffectedProperties[] = {
```

Comment:
marking properties as "zoom effected" in CSSProperties.json should generate table and we should read from it here